pkgs/development/libraries/relibc/default.nix

```diff
@@ -1,27 +1,72 @@
-{ stdenvNoCC, buildPackages, fetchurl }:
+{ stdenvNoCC, buildPackages, makeRustPlatform }:
 
-stdenvNoCC.mkDerivation {
-  name = "binary-relibc-latest";
+let
+  rpath = stdenvNoCC.lib.makeLibraryPath [
+    buildPackages.stdenv.cc.libc
+    "$out"
+  ];
+  bootstrapCrossRust = stdenvNoCC.mkDerivation {
```

Comment:
I would normally be pedantic and say this should be `stdenvNoCC.mkDerivation` (and then you could probably just use `autoPatchelfHook` too), but I think this is about to be replaced in the other PRs so I am not oo concerned :).

Feel free to open another PR if you think this might last more than a few days, though.

Reply:
> I would normally be pedantic and say this should be stdenvNoCC.mkDerivation

I assume that's a typo, because it's stdenvNoCC.mkDerivation already?